src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -201,9 +196,14 @@
         Delimited(Ref("ColumnDatatypeSegment")),
         Ref("UDFParameterDotSyntaxSegment"),
     ),
-    FunctionScriptTerminatorSegment=NamedParser(
-        "function_script_terminator", CodeSegment, type="function_script_terminator"
-    ),
+    # FunctionScriptTerminatorSegment=SegmentGenerator(
```

Comment:
Can this be removed?

Reply:
It can be, but it'll probably want re-adding when it's something we can support